.ruby-version

```diff
@@ -0,0 +1 @@
+3.1.0
```

Comment:
Don't add your local configuration. Please remove this.

Reply:
Updated the changes